go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	github.com/go-redis/redis/v8 v8.11.5
 	github.com/golang-jwt/jwt v3.2.2+incompatible
 	github.com/golang/protobuf v1.5.2
-	github.com/google/go-containerregistry v0.7.1-0.20211214010025-a65b7844a475
+	github.com/google/go-containerregistry v0.8.1-0.20220209165246-a44adc326839
```

Comment:
Do we need to bump the version?

Reply:
No, we don't.
I updated go.mod unintentionally. I checked again, and go.mod needs no modifications.